### DISCUSSION TITLE
Can't login

### DISCUSSION THREAD

**Discussion by mcconnon12** [swellstores/community#177] [Help] [answered]
Hi, I cannot login.

Getting some google recaptcha failed error. Help.

**Comment by ericingram** [comment]
Is it still happening?

**Reply by Strongeyed** [reply]
I have had this issue for months now, glad others are picking up on it. For me I have to click the login button heaps of times and it will eventually login. Some days it more clicks than other days not sure what's going on.

![Screenshot 2023-03-17 at 11 15 12 AM](https://user-images.githubusercontent.com/128098520/225764592-d2e0572c-41de-4575-835f-172e9fc48d85.png)

**Reply by ericingram** [reply]
Sounds like an issue with our "recaptcha" implementation. Sorry for the trouble, but thanks for the feedback and we'll look more into it!

**Reply by MaxBrych** [reply]
have the same issue

**Reply by MaxBrych** [reply]
<img width="1080" alt="image" src="https://user-images.githubusercontent.com/90130774/226093613-e53383bd-ace6-438d-85bc-71b842407100.png">

I can't login

**Reply by mejaz** [reply]
any ETA? i still see this error.

**Reply by mcconnon12** [reply]
It's back again... Just a heads up to everyone who has this... If you have an order email from Swell you should be able to access your Swell account. That has worked for me. Click on the view order details and hopefully that works for you.

**Reply by steven-k1** [reply, marked as answer]
We have made an adjustment to our "recaptcha" implementation that should correct the issue. We will continue to investigate the cause further.

If anyone is still experiencing the issue, please let us know.

**Reply by MaxBrych** [reply]
everything is working again. all good. thank you guys!

**Comment by mcconnon12** [comment]
Hey Eric,

It was happening this morning but I just checked after seeing your message
and was able to login.

Just a heads up I also was not able to charge a customer and I am able to
do so now. Not sure if the issue was related but figure I'd pass along.

Thanks.

On Wed, Mar 15, 2023 at 11:03 AM Eric ***@***.***> wrote:

> Is it still happening?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/swellstores/discussions/177#discussioncomment-5322963>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4ORACKJWI3TM2T4YH4LUU3W4HK3DANCNFSM6AAAAAAV3A5VTA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Best,
Shawn
upglove.com

**Reply by ericingram** [reply]
Appreciate the feedback @mcconnon12!

**Reply by mcconnon12** [reply]
Sure thing Eric. Any chance you have an ETA on when the Horizon theme will hit the hosted Swell stores? Thanks.

**Reply by ericingram** [reply]
@mcconnon12 we've had to re-prioritize some things and don't expect to support hosted Horizon this year. Some major (perhaps more important) changes are coming, and we still plan to get back to this approach in the future. No specific ETA yet.

**Reply by mcconnon12** [reply]
@ericingram Well if you can do anything to push it up we'd appreciate it. We've been waiting to move everything over after the rollout of Horizon. Do any of the new changes gives us additional options with the current store front theme?
